core/rawdb/database.go

```diff
@@ -463,7 +463,10 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 		tds             stat
 		numHashPairings stat
 		hashNumPairings stat
-		tries           stat
+		accountTries    stat
+		storageTries    stat
+		legacyTries     stat
+		stateLookups    stat
```

Comment:
Perhaps reorder these a bit, have the legary ctuff in front, then the pathdb stuff, move state lookups above account/storage nodes. (same in the code below too FWIW).